dlc-manager/src/contract_updater.rs

```diff
@@ -655,7 +656,14 @@ where
         channel_id,
     };
 
-    Ok((signed_contract, fund_psbt.extract_tx()))
+    let transaction = fund_psbt.extract_tx().map_err(|e| {
+        Error::InvalidState(format!(
+            "Could not extract transaction from funding psbt. {}",
+            e
+        ))
+    })?;
```

Comment:
This should fix the error.
```suggestion
        let transaction = fund_psbt.extract_tx_unchecked_fee_rate();
```